ballerina/http_request.bal

```diff
@@ -541,7 +544,7 @@ public class Request {
         } else if payload is mime:Entity[] {
             self.setBodyParts(payload);
         } else if payload is anydata {
-            self.setJsonPayload(payload.toJson());
+            self.setJsonPayload(jsondata:toJson(payload));
```

Comment:
Shall we have a separate function for anydata values? Using `setJsonPayload` does the conversion again which is redundant.

```ballerina
isolated function setAnydataAsJsonPayload(anydata payload) {
     mime:Entity entity = self.getEntityWithoutBodyAndHeaders();
     setJson(entity, jsondata:toJson(payload), self.getContentType(), contentType);
     self.setEntityAndUpdateContentTypeHeader(entity);
}
```

Reply:
Yeah , I will make it separate, then are we going to use `contentType` value as default value ?